chain/get_ancestors.go

```diff
@@ -70,12 +74,11 @@ func GetRecentAncestors(ctx context.Context, base types.TipSet, chainReader Read
 	// This is needed because CollectTipSetsOfHeightAtLeast necessarily reads out
```

Comment:
Comment is still valid if we move to an iterator as suggested below (we can't use the same iterator as it will read past by one).  If we keep it like this the comment should be dropped but the code fine.